ablySpec/RealtimeClientChannel.swift

```diff
@@ -290,6 +290,46 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6i
+                context("expect either") {
+
+                    pending("an array of Message objects") {
+
+                        let client = ARTRealtime(options: AblyTests.commonAppSetup())
+                        defer { client.close() }
+                        let channel = client.channels.get("test")
+
+                        let messages = [ARTMessage(data: ["value":1], name: nil), ARTMessage(data: ["value":2], name: nil)]
+                        var result: [String]?
+
+                        channel.subscribe { message, errorInfo in
+                            result = message.data as? [String]
+                        }
+
+                        channel.publish(messages, callback: nil)
+
+                        expect(result).toEventually(equal(messages), timeout: testTimeout)
```

Comment:
I am not quite sure this is doing what is required. You need to check that both messages are received (which you may be doing, but I'm not quite sure how `messages` will ever equal `result` given you are simply assigning each message's data to the `result` variable), but you also need to check that a single `ProtocolMessage` is being used to send the messages to Ably which you are not doing.